### ISSUE TITLE
Paint Mode is not working in Thunderbird 78

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#111]
The "Paint Mode" command doesn't generate a dropdown menu to select a palette entry / end paint mode:
![image](https://user-images.githubusercontent.com/6314674/107941531-bef31c80-6f81-11eb-90ce-a67c5d1769de.png)

Ibserved Behavior:
When clicked again, it only displays an empty menu square:
![image](https://user-images.githubusercontent.com/6314674/107941568-d03c2900-6f81-11eb-96e8-22e4f41aa162.png)

Expected behavior - a palette menu should be displayed:
![image](https://user-images.githubusercontent.com/6314674/107941620-e34ef900-6f81-11eb-96dd-0d93fb94cd03.png)

yet another regression from the upgrade to Thunderbird 78

**Comment by RealRaven2000** [comment]
Here is a quickfix version - fixes the color menu and also the icons.

[QuickFolders-wx-5.3pre43.zip](https://github.com/RealRaven2000/QuickFolders/files/5981808/QuickFolders-wx-5.3pre43.zip)

to install, download the zip file and drag into Thunderbird Add-ons manager.

**Comment by macdeport** [comment]
At least here, disappearance of _Recent Folders_ tab...
![floating-screenshot-164717](https://user-images.githubusercontent.com/26068595/107968705-f9bf7980-6fae-11eb-8084-b0db2df2fee4.jpg)

**Comment by RealRaven2000** [comment]
> At least here, disappearance of _Recent Folders_ tab...
> ![floating-screenshot-164717](https://user-images.githubusercontent.com/26068595/107968705-f9bf7980-6fae-11eb-8084-b0db2df2fee4.jpg)

Good catch - I did some diffing, it happened between pre37 and pre40 - I tried to fix some coloring issue with JS code. Now it's a pure CSS solution.
![image](https://user-images.githubusercontent.com/6314674/107986424-ecf95080-6fc3-11eb-8a5b-0539aa5ea7ee.png)
Note the paintbrushes having same color as the text.

Here is the new version:

[QuickFolders-wx-5.3pre44.zip](https://github.com/RealRaven2000/QuickFolders/files/5983644/QuickFolders-wx-5.3pre44.zip)
